pymongo/uri_parser.py

```diff
@@ -408,6 +426,11 @@ def parse_uri(uri, default_port=DEFAULT_PORT, validate=True, warn=False,
           wait for a response from the DNS server.
         - 'srv_service_name` (optional): A custom SRV service name
 
+    .. versionchanged:: 4.0
+       Reserved characters that are sub-delimiters ("!", "$", "&", "'",
+       "(", ")", "*", "+", ",", ";", "=") as per RFC 3986 need not be
+       escaped. However, "%" does need to be escaped.
```

Comment:
> Reserved characters that are sub-delimiters ("!", "$", "&", "'",
>       "(", ")", "*", "+", ",", ";", "=") as per RFC 3986 need not be
 >      escaped.
 
 This was already the case except for +. Let's only mention what's actually changed. Perhaps something like this:
 ```
     .. versionchanged:: 4.0
       To better follow RFC 3986, unquoted percent signs ("%") are no longer
       supported and plus signs ("+") are no longer decoded into spaces (" ")
       when decoding username and password. To avoid these issues, use
       :py:meth:`urllib.parse.quote` when building the URI.
```

Reply:
Done.